File: sample/src/main/java/contacts/sample/cheatsheet/other/java/ContactDefaultDataActivity.java
package contacts.sample.cheatsheet.other.java;

import android.app.Activity;

import contacts.core.ContactsFactory;
import contacts.core.entities.Phone;
import contacts.core.util.DefaultContactDataKt;

public class ContactDefaultDataActivity extends Activity {

    Boolean isPhoneTheDefaultPhone(Phone phone) {
        return phone.isDefault();
    }

    Boolean setPhoneAsDefault(Phone phone) {
        return DefaultContactDataKt.setAsDefault(phone, ContactsFactory.create(this));
    }

    Boolean clearDefaultPhone(Phone phone) {
        return DefaultContactDataKt.clearDefault(phone, ContactsFactory.create(this));
    }
}

File: sample/src/main/java/contacts/sample/cheatsheet/other/kotlin/ContactDefaultDataActivity.kt
package contacts.sample.cheatsheet.other.kotlin

import android.app.Activity
import contacts.core.Contacts
import contacts.core.entities.Phone
import contacts.core.util.*

class ContactDefaultDataActivity : Activity() {

    fun isPhoneTheDefaultPhone(phone: Phone): Boolean = phone.isDefault

    fun setPhoneAsDefault(phone: Phone): Boolean = phone.setAsDefault(Contacts(this))

    fun clearDefaultPhone(phone: Phone): Boolean = phone.clearDefault(Contacts(this))
}